src/core/common/Breadcrumbs.js

```diff
@@ -10,21 +11,51 @@ const styles = {
         alignItems: "center",
         border: "1px solid #e5e5e5",
         paddingLeft: "10px",
+        backgroundColor: "white",
+    },
+    separator: {
+        width: "0px",
+        height: "0px",
+        borderTop: "5px solid transparent",
+        borderBottom: "5px solid transparent",
+        borderLeft: "5px solid black",
+        marginLeft: "8px",
+        marginRight: "8px",
+        marginTop: "5px",
+    },
+    link: {
+        textDecoration: "none",
+        color: "#2196f3",
+    },
+    breadcrumbsItem: {
+        display: "flex",
     }
 };
 
-const Breadcrumbs = ({ classes, location }) => {
+/**
+ * This component returns breadcrumbs block
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @param {shape} classes
+ * @param {shape} resource
+ */
+const Breadcrumbs = ({ classes, resource }) => {
     return (
         <div className={classes.breadcrumbsBlock}>
-            <Typography>
-                Home
-            </Typography>
-            <Typography>
-             -
-            </Typography>
-            <Typography>
-                Patient Summary
-            </Typography>
+            <Link to="/charts" className={classes.link} color="inherit">Home</Link>
+            {
+                resource.map(item => {
+                    return (
+                        <div className={classes.breadcrumbsItem}>
```

Comment:
missing `key` prop in iterable component

Reply:
Added